clients/privacy-center/components/modals/privacy-request-modal/PrivacyRequestForm.tsx

```diff
@@ -80,13 +89,21 @@ const usePrivacyRequestForm = ({
 
       // extract identity input values
       const identityInputValues = Object.fromEntries(
-        Object.entries(action.identity_inputs ?? {}).map(([key, field]) => {
-          const value = fallbackNull(values[key]);
-          if (typeof field === "string") {
-            return [key, value];
-          }
-          return [key, { label: field.label, value }];
-        })
+        Object.entries(action.identity_inputs ?? {})
+          // we have to support name as an identity_input for legacy purposes
+          // but we ignore it since it's not unique enough to be treated as an identity
+          .filter(([key]) => key !== "name")
```

Comment:
The `name` field will appear in the form but be ignored from the privacy request payload

Reply:
👍 

OK, this'll work for our purposes. It's pretty confusing behaviour to explain to someone new to the code, but it's also confusing as is, so this is an improvement!